MCD migration - Update SFBO phase description (#4979)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/ms-cloud-germany-transition-phases.md

- Contacts and meetings will be migrated to Microsoft Teams.
- Users won't be able to sign in to Skype for Business between time service transitions to Office 365 services, and not until customer DNS entries are completed.
- Contacts and existing meetings will continue to function as Skype for Business meetings.

When a vanity domain has been configured for Skype for Business, the DNS entries must be updated. Please refer to [Domains in the Microsoft 365 admin center](https://admin.microsoft.com/Adminportal/Home#/Domains) and apply the changes in your DNS configuration. 

If you have to connect to Skype for Business Online with PowerShell after migration phase 9 has been completed, use the following PowerShell code to connect:

```powershell
Import-Module MicrosoftTeams
$userCredential = Get-Credential
Connect-MicrosoftTeams -Credential $userCredential
```

### Known limitations until finalizing Azure AD migration

> Microsoft Teams is leveraging features of Azure AD. While the migration of Azure AD is not completed, some features of Microsoft Teams are not completely available. After phase 9, when the migration of Azure AD will be finalized, the following features become fully available:

Reviewer: ```suggestion
Microsoft Teams is leveraging features of Azure AD. While the migration of Azure AD is not completed, some features of Microsoft Teams are not fully available. After phase 9, when the migration of Azure AD has been finalized, the following features become fully available:
```

- Apps cannot be managed in the Microsoft Teams admin center.

> - New Teams can be created in the Microsoft Teams client only unless the Teams administrator constrained permissions to create new Teams for users. New Teams cannot be created in the Microsoft Teams admin center. 

Reviewer: ```suggestion
- New teams can be created in the Microsoft Teams client only unless the Teams administrator has limited the permissions for users to create new teams. New teams cannot be created in the Microsoft Teams admin center. 
```

> - The web version of Microsoft Teams (teams.microsoft.com) is not available.

Reviewer: ```suggestion
- The web version of Microsoft Teams is not available.
```
